go/ql/lib/ext/github.com.sirupsen.logrus.model.yml

```
@@ -0,0 +1,131 @@
+extensions:
+  - addsTo:
+      pack: codeql/go-all
+      extensible: packageGrouping
+    data:
+      - ["logrus", "github.com/sirupsen/logrus"]
+      - ["logrus", "ggithub.com/Sirupsen/logrus"]
```

Typo in package path
```suggestion
      - ["logrus", "github.com/Sirupsen/logrus"]
```